velox/connectors/hive/HiveConnector.cpp

```diff
@@ -72,6 +72,9 @@ std::string HiveTableHandle::toString() const {
     }
     out << "]";
   }
+  if (remainingFilter_) {
+    out << ", remaining filters: (" << remainingFilter_->toString() << ")";
```

Comment:
Since there is only one filter, perhaps, change the label to "remaining filter".